client/src/components/Pagination.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+const Pagination = (props)=>{
+const {onPaginate} = props;
+const {paginate} = props;
+paginate.propTypes = {
```

Comment:
Proptypes validation should be outside function? Also you mean `Pagination.propTypes` instead?